notebooks/02-Introduction.ipynb

```diff
@@ -0,0 +1,233 @@
+{
```

Comment:
<p>The indentation in the list does not come out correct.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/asdf-format/scipy2022tutorial/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Could you clarify this?</p>